### PR TITLE
Add conda install instructions

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

**Comment by tribeiro** [comment]
@couger01, you need to rebase this branch.

File: doc/index.rst [hunk 1]
```diff
@@ -25,6 +25,24 @@ Using lsst.ts.salobj
     sal_scripts
     :maxdepth: 2
 
+
+Conda Install
+-------------
+ts_salobj is offered as a conda package.
```
Comment (tribeiro) [inline doc/index.rst:31]: Note that we also offer salobj in pypi, so you can actually do `pip install ts-salobj` as well.

**Review by tribeiro — CHANGES_REQUESTED** [review]
This is a good start but I think it would be good to expand this with all the information needed to install salobj on centos. From installing OpenSplice with yum, all the way to having salobj up and running, both with conda and pypi.

**Review by pkubanek — APPROVED** [review]

**Review by pkubanek — APPROVED** [review]
I wrote something on similar tone for installing, but never merged it. Thanks for providing this missing piece.

**Pull request closed without merging**